src/mame/drivers/missb2.cpp

```diff
@@ -16,15 +16,13 @@ written, so it may be normal behaviour.
 
 #include "emu.h"
 #include "includes/bublbobl.h"
-
```

Comment:
I didn't realize there was a significance to it (I had thought it was an old c/p error from years ago), I'll put it back.